ArmVirtPkg/ArmVirtCloudHv.dsc

```diff
@@ -201,9 +201,6 @@
 [PcdsDynamicHii]
   gUefiOvmfPkgTokenSpaceGuid.PcdForceNoAcpi|L"ForceNoAcpi"|gOvmfVariableGuid|0x0|FALSE|NV,BS
 
-[PcdsPatchableInModule.common]

```

Comment:
Should this be commented out with a MU_CHANGE instead?

Reply:
Multiple other files below that just delete. I don't mind which we do. just bringing it to your attention :)

Reply:
This actually was directly from edk2. https://github.com/tianocore/edk2/commit/7bcd49edd0a1daa23f8e910d0082abb0f9f2119d.

Reply:
I'll add a `CHERRY-PICK` prefix to that commit.